depinject.go

```diff
@@ -96,14 +96,10 @@ func ProvideModule(in ModuleInputs) ModuleOutputs {
 	}
 }
 
-type ComponentsBankKeeper interface {
-	actiontypes.BankKeeper
-}
-
 type ComponentsInputs struct {
 	Orbiters *keeper.Keeper
 
-	BankKeeper ComponentsBankKeeper
+	BankKeeper bankkeeper.Keeper
```

Comment:
I'm fine with this change, just wanted to understand if there was a specific reason to remove the abstracted interface here?